### DISCUSSION TITLE
Track Logged in user session

### DISCUSSION THREAD

**Discussion by Daniel-Alamezie** [swellstores/community#422] [Help] [answered]
I am sure there is probably something i am missing but i would appreciate the response and help here.
I am looking to track logged in users from my front app, to make sure logged in users have access to certain priviledges.

Now i know i can do this with session token, and i presume swell provides one when a user logs in, but i dont know which one of these does that Job if any even does that job
![image](https://github.com/swellstores/community/assets/74880857/352f3b00-7af7-4db4-bdac-e9ec3a3d33b0)

I will appreciate the swift respone

Thanks

**Comment by ericingram** [comment, marked as answer]
If a user is logged in, you can call `swell.account.get()` anywhere in your app and use the result to track things

The session cookie itself may exist regardless if the user is authenticated or not
